packages/noya-state/src/selectors/layerSelectors.ts

```diff
@@ -26,7 +27,9 @@ export const getSelectedTextLayers = (
   );
 };
 
-export const getSelectedLayers = (state: ApplicationState): PageLayer[] => {
+export const getSelectedLayers = (
+  state: ApplicationState | Draft<ApplicationState>,
```

Comment:
I think just `Draft<ApplicationState>` will be enough here. It seems like that type also encompasses the non-draft type